hub/core/dataset.py

```diff
@@ -0,0 +1,10 @@
+from hub.core.typing import StorageProvider
```

Comment:
This looks like a helper methods, can we rename this as such?

Reply:
not sure what you mean by this.